extensions/shared/plan-upgrade-notification.js

```diff
@@ -15,9 +21,35 @@ if ( undefined !== typeof window && window.location ) {
 	const { query } = parseUrl( window.location.href, true );
 
 	if ( query.plan_upgraded ) {
-		dispatch( 'core/notices' ).createNotice(
-			'success',
-			__( 'Plan upgraded. You may use premium blocks now.', 'jetpack' )
-		);
+		const path = isSimpleSite() ? `/rest/v1.2/sites/${ window.location.host }` : '/jetpack/v4/site';
+
+		// apiFetch.use( apiFetch.createRootURLMiddleware( 'https://public-api.wordpress.com/rest/v1.2' ) );
+
+		apiFetch( { path } )
```

Comment:
For simple sites, the api call still ends up being made to `wp/v2` e.g. 

```
https://public-api.wordpress.com/wp/v2/sites/165510172//rest/v1.2/sites/chriskmnds.wordpress.com?_envelope=1&_locale=user
```

which looks odd. I must be missing something obvious here. I have tried with `createRootURLMiddleware` to force the root url, but to no avail.